docs/sites.yml

```diff
@@ -5361,6 +5361,15 @@
     - Space
   built_by: Jonathan Z. Fisher
   built_by_url: "https://jonzfisher.com"
+- title: Sal's Pals
+  main_url: https://www.sals-pals.net
+  url: https://www.sals-pals.net
+  featured: false
+  description: >
+    Sal's Pals is a professional dog walking and pet sitting service based in Westfield, NJ. New site built with gatsby v2.
+  categories:
+    - Dog walking
+    - Pet sitting
```

Comment:
I think those categories (that don't exist right now) are too specific - ideally some already existing one could be reused, or if not at least more general would be created